tasks/push-to-cdn/push-to-cdn.yaml

```diff
@@ -0,0 +1,110 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: push-to-cdn
+  labels:
+    app.kubernetes.io/version: "4.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push content to Red Hat's CDN
+  params:
+    - name: releasePlanAdmission
+      type: string
+      description: The namespaced name of the ReleasePlanAdmission
+      default: "rhtap-releng-tenant/rpa" # This needs to change later
+    - name: subdirectory
+      description: Subdirectory inside the workspace to be used
+      type: string
+      default: ""
+  workspaces:
+    - name: data
+      description: Workspace to save the CR jsons to
+  results:
+    - name: releasePlanAdmission
+      type: string
+      description: The relative path in the workspace to the stored releasePlanAdmission json
+  steps:
+    - name: create-exodus-conf-file
+      image: quay.io/distribution_relengs/publish-to-dev-portal-test-image
```

Comment:
You will use the release-service-utils image in this task, right? Same with the other step.

Reply:
Yes, For testing purposes I have added this image which is a clone of release-service-utils. I update that before the MR is ready to merge!

Reply:
OK, sounds good, thanks.

Reply:
The new image is available here quay.io/konflux-ci/release-service-utils:38c3bfd7479c86b832cba5b61f9bbde40c469393